### PR TITLE
Fix coding style

### PR DESCRIPTION
**Pull request by t-rufang**

### CODE REVIEW

**Comment by andxu** [comment]
I need to reopen this PR to add checkstyle check

**Comment by andxu** [comment]
Please fix it:
image

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobLocalRunConfigurationProducer.java [hunk 1]
```diff
@@ -91,59 +100,66 @@ private void setupConfiguration(LivySparkBatchJobRunConfiguration configuration,
...
+    private static Optional<String> getNormalizedClassName(@NotNull final PsiClass clazz) {
         return Optional.ofNullable(SparkContextUtilsKt.getNormalizedClassNameForSpark(clazz));
     }
 
     /**
      * The function to help reuse RunConfiguration
+     *
      * @param jobConfiguration Run Configuration to test
-     * @param context current Context
+     * @param context          current Context
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobLocalRunConfigurationProducer.java:111]: No need for these spaces

Comment (t-rufang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobLocalRunConfigurationProducer.java:111]: This change happened after I do auto-format in IntelliJ. The space can make the description of the parameter aligned vertically.
image

**Pull request merged**